plugin/src/main/kotlin/tanvd/grazi/language/Lang.kt

```diff
@@ -47,6 +48,8 @@ enum class Lang(val displayName: String, val shortCode: String, private val clas
         get() {
             if (_jLanguage == null) {
                 _jLanguage = GraziPlugin.loadClass("org.languagetool.language.$className")?.newInstance() as Language?
+
+                if (_jLanguage != null) enableLatinLettersInSpellchecker(this)
```

Comment:
I would prefer to use object name explicitly.

Reply:
Done